### PR TITLE
rename controller operator resource instance from 'instance' to 'cluster'

### PR DESCRIPTION
**Pull request by bparees**
makes us consistent w/ the other controller+api operators.

### CODE REVIEW

**Comment by bparees** [comment]
@deads2k you're welcome

/assign @adambkaplan

**Comment by bparees** [comment]
> nit - we should also update the operator progressing message here: https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L104

Good catch, updated.

File: pkg/operator/sync_openshiftcontrollermanager_v311_00.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func syncOpenShiftControllerManager_v311_00_to_latest(c OpenShiftControllerManag
 		progressingMessages = append(progressingMessages, fmt.Sprintf("daemonset/controller-manager: observed generation is %d, desired generation is %d.", actualDaemonSet.Status.ObservedGeneration, actualDaemonSet.ObjectMeta.Generation))
 	}
 	if operatorConfig.ObjectMeta.Generation != operatorConfig.Status.ObservedGeneration {
-		progressingMessages = append(progressingMessages, fmt.Sprintf("openshiftcontrollermanageroperatorconfigs/instance: observed generation is %d, desired generation is %d.", operatorConfig.Status.ObservedGeneration, operatorConfig.ObjectMeta.Generation))
+		progressingMessages = append(progressingMessages, fmt.Sprintf("openshiftcontrollermanageroperatorconfigs/cluster: observed generation is %d, desired generation is %d.", operatorConfig.Status.ObservedGeneration, operatorConfig.ObjectMeta.Generation))
```
Comment (adambkaplan) [inline pkg/operator/sync_openshiftcontrollermanager_v311_00.go]: @bparees I think the CRD for the operator config is now is now `openshiftcontrollermanagers.operator.openshift.io` - should update this message so an admin can copy/paste to get the details.

Comment (bparees) [reply pkg/operator/sync_openshiftcontrollermanager_v311_00.go]: right. updated.

**Comment by adambkaplan** [comment]
/retest

Install failed - hung deploying networking

**Comment by bparees** [comment]
@adambkaplan this is passing tests now

**Pull request merged**